backend/config/config.go

```diff
@@ -124,3 +126,20 @@ func (c *Config) openNewDBConnection() (*gorm.DB, error) {
 
 	return gorm.Open("postgres", dbstring)
 }
+
+// add user config
+func (c *Config) SetUserConfig(userConfig *TokenInfo) *Config {
```

Comment:
We should have a consistent in naming (variable should have some thing related with it type or have a appropriate context)